portal/static/js/psaTracker.js

```diff
@@ -337,7 +347,10 @@
                         var tempResults = results;
                         results = results.slice(0, 10);
                         self.history.items = tempResults.slice(10, 20);
-                        self.history.sidenote = String(i18next.t("* Ten results since {year}")).replace("{year}", self.getHistoryMinYear());
+                        self.history.sidenote = String(self.i18next.t("* Ten results since {year}")).replace("{year}", self.getHistoryMinYear());
```

Comment:
If the number of results may change, it might be better to use a placeholder (and `10` vs `ten`) so we don't need to update the translation later